staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -80,20 +80,25 @@ type fakeObjectDefaulter struct{}
 func (d *fakeObjectDefaulter) Default(in runtime.Object) {}
 
 type TestFieldManager struct {
-	fieldManager fieldmanager.Manager
+	fieldManager *fieldmanager.FieldManager
 	emptyObj     runtime.Object
 	liveObj      runtime.Object
 }
 
-func NewTestFieldManager(gvk schema.GroupVersionKind) TestFieldManager {
+func NewTestFieldManager(gvk schema.GroupVersionKind, managers ...func(fieldmanager.Manager) fieldmanager.Manager) TestFieldManager {
```

Comment:
I don't know if you need more than one, since you would be able to chain them yourself?

Reply:
Oh yep, removed